README.md

```diff
@@ -4,24 +4,26 @@ The toffy repo is designed to simplify the process of generating and processing
 This repo is currently in beta testing. None of the code has been published yet, and we will be making breaking changes frequently. If you find bugs, please [open an issue](https://github.com/angelolab/toffy/issues/new/choose). If you have questions or want to collaborate, please reach out to Noah (nfgreen@stanford.edu)
 
 ## Table of Contents
-- [Overview](#overview)
-  - [1. Using toffy for the first time](#1-using-toffy-for-the-first-time)
-  - [2. Setting up a MIBI run](#2-setting-up-a-mibi-run)
-  - [3. Evaluating a MIBI run](#3-evaluating-a-mibi-run)
-  - [4. Processing data after a MIBI run](#4-processing-mibi-data)
-  - [5. Formatting data for downstream analysis](#5-formatting-mibi-runs-for-analysis)
-- [Installation](#installation)
-  - [Requirements for specific operating systems](#requirements-for-specific-operating-systems)
-    - [Windows](#windows)
-    - [macOS](#macos)
-  - [Setting up the virtual environment](#setting-up-the-virtual-environment)
-  - [Using the repo](#using-the-repo)
-  - [Updating the repo](#updating-the-repo)
-- [Panel format](#panel-format)
-- [Questions?](#questions)
+- [toffy](#toffy)
```

Comment:
Adding a hyperlink to toffy and ToC makes it less readable imo, as now everything has an extra level of indentation. In addition, it doesn't really serve a purpose, as no one is going to need to use this ToC to navigate to the description at the top, or the ToC itself.